pytorch_lightning/trainer/optimizers.py

```diff
@@ -86,8 +87,10 @@ def _convert_to_lightning_optimizer(trainer, optimizer):
             optimizer._on_trainer_init(trainer)
             return optimizer
 
-        if self._enable_pl_optimizer:
-            self.optimizers = [_convert_to_lightning_optimizer(self, opt) for opt in self.optimizers]
+        self._lightning_optimizers = OrderedDict({
```

Comment:
Is `OrderedDict` instead of `dict` necessary?

Reply:
I think OrderedDict are slightly faster, but it is not necessary. Let me remove it.